DataProcessor/DataUnpacker.cpp

```diff
@@ -38,7 +38,7 @@ E bytesToGeneralData(QByteArray data, int startPos, int endPos, E typeZero)
 
 DataUnpacker::DataUnpacker(QObject *parent) : QObject(parent)
 {
-    FILE* fp = fopen("../sc1-driver-io/sc1-data-format/format.json", "r"); // NOTE: Windows: "rb"; non-Windows: "r"
+    FILE* fp = fopen("../solar-car-dashboard/sc1-data-format/format.json", "r"); // NOTE: Windows: "rb"; non-Windows: "r"
```

Comment:
```suggestion
    FILE* fp = fopen("../sc1-driver-io/sc1-data-format/format.json", "r"); // NOTE: Windows: "rb"; non-Windows: "r"
```
This needs to be sc1-driver-io so that the project will run correctly for members who cloned the repo after it was renamed. If the project name is solar-car-dashboard for a member, that is handled by the next three lines